.pipelines/daily_addons_enablement_test.yaml

```diff
@@ -0,0 +1,64 @@
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily midnight test
+  branches:
+    include:
+    - ci_prod
+jobs:
+    - job: test
+      variables:
+        armServiceConnectionName: 'ci-1es-acr-connection'
+        subscription: '9b96ebbd-c57a-42d1-bbe9-b69296e4c7fb'
+      pool:
+        name: Azure-Pipelines-CI-Test-EO
+      steps:
+          - task: AzureCLI@2
+            inputs:
+              azureSubscription: '${{ variables.armServiceConnectionName }}'
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                az --version
+                az account show
+                az account set -s ${{ variables.subscription }}
+                
+                # Check if the omsagent addon is enabled
+                addonEnabled=$(az aks show --resource-group $(RESOURCEGROUP) --name $(AKSCLUSTERNAME) --query addonProfiles.omsagent.enabled -o tsv)
+                
+                if [ "$addonEnabled" == "true" ]; then
+                    echo "OMSAgent addon is enabled on the AKS cluster."
+                    disable=$(az aks disable-addons --addons monitoring --name $(AKSCLUSTERNAME) --resource-group $(RESOURCEGROUP))
+                    omsagentEnabled=$(echo $disable | jq '.addonProfiles.omsagent.enabled')
+                    if [ $omsagentEnabled == "true" ]; then
+                      echo "Omsagent is not disabled."
+                      exit -1
+                    fi
+                    
+                    echo "Re-enabling the OMSAgent addon..."
+                    enable=$(az aks enable-addons --addons monitoring --name $(AKSCLUSTERNAME) --resource-group $(RESOURCEGROUP) --workspace-resource-id $(LOGANALYTICSWORKSPACE))
+                    omsagentEnabled=$(echo $enable | jq '.addonProfiles.omsagent.enabled')
+                    if [ $omsagentEnabled == "false" ]; then
+                      echo "Omsagent is not enabled"
+                      exit -1
+                    fi
+                    omsagentWorkspace=$(az aks show --resource-group $(RESOURCEGROUP) --name $(AKSCLUSTERNAME) --query addonProfiles.omsagent.config.logAnalyticsWorkspaceResourceID -o tsv)
+                    if [ "$omsagentWorkspace" != "$(LOGANALYTICSWORKSPACE)" ]; then
+                      echo "Omsagent is not enabled with the correct log analytics workspace - $omsagentWorkspace"
+                      exit -1
+                    fi
+                    
+                    echo "Sleeping for everything to get set up and logs start flowing"
+                    sleep $(SLEEPDURATION)

```

Comment:
SLEEPDURATION has to be greater than 5m which is query interval and otherwise we will be querying the data before disable.  whats the default value of SLEEPDURATION  pipeline variable?

Reply:
It is greater than 5 min. It is 6mi